Update pack versions (#1093)
spectrocloud/librarium: content/docs/06-integrations/00-kong.md

---

import WarningBox from 'shared/components/WarningBox';
import Tabs from 'shared/components/ui/Tabs';
import InfoBox from 'shared/components/InfoBox';
import PointsOfInterest from 'shared/components/common/PointOfInterest';
import Tooltip from "shared/components/ui/Tooltip";

# Kong Ingress Controller

> The Kong integration is an Ingress Controller for Kubernetes that configures ingress with a load balancer. Kong can also configure your edge router or additional frontends to help handle incoming requests.

Reviewer: ```suggestion
The Kong integration is an Ingress Controller for Kubernetes that configures ingress with a load balancer. You can use the Kong as an application load balancer for your application.
```
The later statement is confusing so let's just remove it.

## Version Supported

<Tabs>

<Tabs.TabPane tab="2.13.x" key="2.13.x">

* **2.13.1**

</Tabs.TabPane>

<Tabs.TabPane tab="1.4.x" key="1.4.x">

* **1.4.0**

</Tabs.TabPane>

</Tabs>

## Components

> The integration adds the Kong Ingress Controller, which exposes a service of type LoadBalancer.

Reviewer: ```suggestion
The integration adds the Kong Ingress Controller, which exposes a Kubernetes service of type LoadBalancer.
```
We use service a bit through our documentation so being specific helps a bit.
